packages/xo-server/src/xapi/mixins/patching.mjs

```diff
@@ -478,7 +478,15 @@ export default {
     throw new Error('non pool-wide install not implemented')
   },
 
-  async rollingPoolUpdate() {
+  @decorateWith(deferrable)
+  async rollingPoolUpdate($defer) {
+    if (this.pool.ha_enabled) {
+      const haSrs = this.pool.$ha_statefiles.map(vdi => vdi.SR)
+      const haConfig = this.pool.ha_configuration
+      await this.call('pool.disable_ha')
+      $defer(() => this.call('pool.enable_ha', haSrs, haConfig))
```

Comment:
I find no documentation on where to find this SRs.

Reply:
What do you mean? I think you can choose any SRs for HA and I'm just re-using the same ones as before the rolling pool update.

Reply:
Ok, if that's the best way we found to get this info, I guess it's good enough.